### DISCUSSION TITLE
No reachable departures

### DISCUSSION THREAD

**Discussion by Pade2204** [KristjanESPERANTO/MMM-PublicTransportHafas#187] [Q&A]
Hi,
I have tried to install the module and now it works. Thanks.

But I get "No reachable departures" when I try many "rejseplanen" IDs.

Example:

```
hafasProfile: "rejseplanen",
stationID: "8603349",
```

Anyone know what I did wrong`?

The StationID you mentioned earlier works, but not the right station. May be central station.

![1](https://github.com/user-attachments/assets/58e597fe-97e9-4dd9-82dc-71f27afa5b41)
![2](https://github.com/user-attachments/assets/fb28b519-a1e4-41b2-96ee-3a37c2551acf)

**Comment by Pade2204** [comment]
Too bad this module does not have buses and metro from rejseplanen profile (Denmark/Copenhagen). S-train and trains seem fine. Thanks anyway.

**Reply by KristjanESPERANTO** [reply]
Too bad. Then you will have to look for a module or write one yourself that uses the other API (which you mentioned in discussion https://github.com/KristjanESPERANTO/MMM-PublicTransportHafas/discussions/174).
